extensions/amp-inline-gallery/amp-inline-gallery.md

```diff
@@ -0,0 +1,225 @@
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# amp-inline-gallery
+
+A carousel for displaying content such as images along a horizontal axis along with optional pagination dots and preview thumbnails.
+
+<table>
+  <tr>
+    <td width="40%"><strong>Availability</strong></td>
+    <td><div><a href="https://amp.dev/documentation/guides-and-tutorials/learn/experimental">Experimental</a>; You must turn on the <code>amp-base-carousel</code> experiment to use this component.</div></td>
+  </tr>
+  <tr>
+    <td width="40%"><strong>Required Scripts</strong></td>
+    <td>
+      <div>
+        <code>&lt;script async custom-element="amp-inline-gallery" src="https://cdn.ampproject.org/v0/amp-inline-gallery.js">&lt;/script></code>
+      </div>
+      <div>
+        <code>&lt;script async custom-element="amp-base-carousel" src="https://cdn.ampproject.org/v0/amp-base-carousel.js">&lt;/script></code>
+      </div>
+    </td>
+  </tr>
+  <tr>
+    <td class="col-fourty"><strong><a href="https://amp.dev/documentation/guides-and-tutorials/develop/style_and_layout/control_layout">Supported Layouts</a></strong></td>
+    <td>
+      container
+    </td>
+  </tr>
+</table>
+
+## Usage
+
+The `<amp-inline-gallery>` component uses an `<amp-base-carousel>` to display slides. Note that scripts to load both extensions are required. Typical usage might look like:
+
+[example preview="inline" playground="true" imports="amp-inline-gallery,amp-base-carousel"]
+
+```html
+<amp-inline-gallery layout="container">
+  <amp-base-carousel
+    class="gallery"
+    layout="responsive"
+    width="3.6"
+    height="2"
+    snap-align="center"
+    loop="true"
+    visible-count="1.2"
+    lightbox
+  >
+    <amp-img
+      src="{{server_for_email}}/static/inline-examples/images/image1.jpg"
+      layout="responsive"
+      width="450"
+      height="300"
+    ></amp-img>
+    <amp-img
+      src="{{server_for_email}}/static/inline-examples/images/image2.jpg"
+      layout="responsive"
+      width="450"
+      height="300"
+    ></amp-img>
+    <amp-img
+      src="{{server_for_email}}/static/inline-examples/images/image3.jpg"
+      layout="responsive"
+      width="450"
+      height="300"
+    ></amp-img>
+  </amp-base-carousel>
+  <amp-inline-gallery-pagination layout="nodisplay" inset>
+  </amp-inline-gallery-pagination>
+</amp-inline-gallery>
+```
+
+[/example]
+
+The above example shows slides using an aspect ratio of 3:2, with 10% of a slide peeking on either side. Note that the carousel itself uses an aspect ratio of 3.6:2 since we show 1.2 slides at a time.
+
+### Using pagination indicators
+
```

Comment:
Insert a section above here:
"Display pagination indicators by including the `<amp-inline-gallery-pagination>` element within the `<amp-inline-gallery">` component." 

Small code sample here please :)